xbmc/addons/addoninfo/AddonType.cpp

```diff
@@ -8,16 +8,23 @@
 
 #include "AddonType.h"
 
+#include "addons/IAddon.h"
 #include "addons/addoninfo/AddonInfo.h"
 #include "utils/URIUtils.h"
 
 namespace ADDON
 {
+
 static const std::set<TYPE> dependencyTypes = {
     ADDON_SCRAPER_LIBRARY,
     ADDON_SCRIPT_LIBRARY,
     ADDON_SCRIPT_MODULE,
 };
+
+static const std::set<TYPE> invisibleAddonTypes = {
+    ADDON_SCRIPT_MODULE,
+};
```

Comment:
I would place this definition in CGUIAddonInfoDialog, because it is logic only needed there.